### DISCUSSION TITLE
MacOS Updates problematic?

### DISCUSSION THREAD

**Discussion by MAUIoxo** [dotnet/maui#25548] [General]
My MacOS is showing a possible update to macOS Sequoia right now.

I don't want to risk any incompatibilities and problems with all the tools in the .NET MAUI Toolchain including Xcode and Command Line Tools.

Therefore, I just wanted to ask in general if there are expected problems/incompatibilities for instance whith Xcode, Command Line Tools or whatever is also involved and could end up in a problem when updating the OS itself (maybe as they use certain libraries coming with the OS)?

**Comment by timdinh-grex** [comment]
I have Xcode 16.0 running on 15.1.1 and developing MAUI blazor hybrid fine

**Comment by MAUIoxo** [comment]
Thanks for your experience on this version combination. In general, are they coupled in any way so that I have to be careful in general when updating Mac OS? Do you or does anybody else know or have experience on that?
